neurodsp/spectral.py

```diff
@@ -439,36 +466,46 @@ def morlet_convolve(x, f0, Fs, w=7, s=.5, M=None, norm='sss'):
     return mwt_real + 1j * mwt_imag
 
 
-def rotate_powerlaw(spectrum, f_axis, delta_f, f_rotation=30):
-    """Takes a PSD and changes its power law exponent about a given axis (frequency).
+def rotate_powerlaw(psd, f_axis, delta_f, f_rotation=None):
```

Comment:
It seems weird to me to have input & output order as 'psd', 'freqs'. I'm used to the other order - but that could be just me. Is there a general order for NDSP? I did notice that this is inconsistent with the psd function, which is annoying. 

For example `rotate_powerlaw(*psd(signal, fs), 0.5, 30)` would not work, because the order will be wrong, and one might want this to work. It would also be nice to have things like this be consistent between modules. 

tl-dr: can we switch input & output order?

Reply:
sure, good with me. Not sure if this comes up anywhere else, but we can make this consistent across.